infrastructure/repository/authzed/SpiceDbAccessRepository_test.go

```diff
@@ -148,7 +148,7 @@ func TestUnassignBatch(t *testing.T) {
 
 	// given
 	subs := []domain.SubjectID{
-		"u1",
+		"u1", "u3",
```

Comment:
fyi, random find while looking at the log messages running the tests (changed logging to error to let them print out in tests, reverted later): this test did not unassign a batch, but unassign one seat, so fixed it.